extra/utils/validation.js

```diff
@@ -14,16 +14,14 @@ const users =
         reviewerEmail: 'megha.r@suessive.tech',
         }
     ];
-count=0;
-count2=0;
 valid=[];
 invalid=[]; 
 function validateEmail(x)
 {
    const regex =  /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((successive.tech))$/;
    y=regex.test(x);
 
-    if(y==true) 
+    if(y) 
         valid.push(x);
     else 
```

Comment:
validateEmail is for validation only it shouldn't update the array. Follow single responsibility principle.